src/data/repositories/common/getProgramIndicatorsFromDatastore.ts

```diff
@@ -0,0 +1,32 @@
+import { Maybe } from "../../../utils/ts-utils";
+import { FutureData } from "../../api-futures";
+import { DataStoreClient } from "../../DataStoreClient";
+
+export enum ProgramIndicatorsDatastoreKey {
```

Comment:
again, the enums and types should not refer to DHIS concepts like  ProgramIndicators, please rename.